src/Conversion/ONNXToMhlo/CMakeLists.txt

```diff
@@ -45,10 +53,10 @@ add_onnx_mlir_library(OMONNXToMhlo
   OMONNXRewrite
   OMONNXOps
   OMSupport
-  )
```

Comment:
Why? We generally follow the pattern of indenting the closing parenthesis, so the change should be the other way around - indent it everywhere.

Reply:
This is what my auto-formatter do, fixed.